module/main.c

```diff
@@ -131,13 +127,13 @@ int16_t output, output_new;
 #define FIRSTRUN_KEY 0x22
 
 typedef const struct {
-    tele_script_t script[10];
-    tele_pattern_t patterns[4];
+    scene_script_t script[10];
```

Comment:
See `state.h`, that's where the actual scene types are defined. This is just for saving data to and from nvram. I haven't paid much attention to it, but it probably should be renamed to `scripts`.

Putting `scene` at the front is an attempt at namespacing. I've renamed `tele_pattern_t` to `scene_pattern_t` too.